DotNetOnion.Tests/MonohopCircuits.cs

```diff
@@ -6,7 +6,7 @@
 
 using NUnit.Framework;
 
-using NOnion;
```

Comment:
The commit message title of this makes me think that it's redundant or confusing, because the lib is already called NOnion. But thinking about this more, I guess we can rename NOnion project/assembly to NOnion.Core? Also, DotNetOnion.Tests to NOnion.Tests

Reply:
The main reason behind that was to declutter the root namespace in this instance being "NOnion" and moving TorStream - Guard- Circuit to a Network folder/namespace

Reply:
Fair enough, let's not rename NOnion project to NOnion.Core, but then don't use it in the commit message titles, and use a subnamespace of the code you're changing (e.g. Network). Also, still let's rename DotNetOnion.Tests to NOnion.Tests

Reply:
Done